packages/frontend/web-ui/src/hooks/useForm.js

```diff
@@ -0,0 +1,99 @@
+/* eslint-disable react-hooks/exhaustive-deps */
+import { useEffect, useMemo, useState } from 'react';
+
+export const useForm = ( initialForm = {} ) => {
```

Comment:
Let's try to avoid a form hook for every component in the page in favor of specific ones

Reply:
Done!